docs/source/selection.rst

```diff
@@ -246,3 +246,51 @@ be instantiated using
     selector.fit(X, y)
 
     Xr = selector.transform(X)
+
+Voronoi FPS
+###########
+In FPS, points are selected based upon their Hausdorff distance to
+previous selections, i.e. the minimum distance between a given point and
+any previously selected points. This implicitly constructs a Voronoi
+tessellation which is updated with each new selection, as each unselected
+point "belongs" to the Voronoi polyhedron of the nearest previous selection.
+
+This implicit tessellation enabled a more efficient evaluation of the FPS --
+at each iteration, we need only consider for selection those points at the
+boundaries of the Voronoi polyhedra, and when updating the tessellation
+we need only consider moving those points whose Hausdorff distance is
+greater than half of the distance between the corresponding Voronoi center
+and the newly selected point, per the triangle equality.
+
+This is particularly appealing when using a non-Euclidean or
+computationally-intensive distance metric, for which the
+decrease in computational time due to the reduction in distance
+calculations outweighs the increase from bookkeeping.
+
+These selectors can be instantiated using :py:class:`skcosmo.sample_selection.VoronoiFPS`.
+
+.. code-block:: python
+
+    from skcosmo.feature_selection import VoronoiFPS
+    selector = VoronoiFPS(
+                        n_to_select=4,
+                        progress_bar=True,
+                        score_threshold=1E-12
+                        full=False,
```

Comment:
You should include any other additional parameters that can be passed to VFPS, namely `n_trial_calculation` and  `full_fraction`

Reply:
Agree, done